src/main/scala/GameHelpers.scala

```diff
@@ -0,0 +1,69 @@
+import scala.util.Random
+
+object GameHelpers {
```

Comment:
I avoid using non specific words such as "Helpers" as all code should be helping with something specific.  It can turn into a bit of a catch all otherwise.  A good name should exclude more stuff than it includes.  The same applies for "common" and "service".
It does make naming a lot harder, because you have to keep thinking of specific names, but it helps you focus on the higher level structure a bit.